src/World.h

```diff
@@ -79,7 +79,7 @@ typedef cItemCallback<cFlowerPotEntity>    cFlowerPotCallback;
 
 
 
-
+const int CHUNK_CHANCE_ACCURACY = 100000;
```

Comment:
The difficulty with doing LRU is that it makes unloading an O(N^2) operation for the naive algorithm. We really don't want to have to do that. What might work is maintaining a doubly linked list in order of time unloaded.